### PR TITLE
fix: add topology mark for test_bgp_peer_shutdown.py

### PR DESCRIPTION
**Pull request by cyw233**
### Description of PR

Add topology mark for `test_bgp_peer_shutdown.py` so it won't get skipped

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

### CODE REVIEW

**Comment by StormLiangMS** [comment]
@cyw233  do we get T2 tested? If not, could you test it before we merge this? Or we can skip on T2 first, when it is ready, we can enable it.

**Comment by cyw233** [comment]
> @cyw233 do we get T2 tested? If not, could you test it before we merge this? Or we can skip on T2 first, when it is ready, we can enable it.

Ah yes, we are in the middle of supporting T2 testbed for most test cases. I haven't run this test on T2 before but I'm sure we are aiming to support it in the future. Let's skip it for now. Thanks!

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**
